src/data/full-quiz.js

```diff
@@ -1,5 +1,60 @@
 const fullQuiz = [
   {
+    Question:
+      "Which of the following is the default separator in join() method in javascript?",

```

Comment:
I think we should add the word "the" between in and join().
Also, please make sure to property capitalize JavaScript.

New sentence: Which of the following is the default separator in the join() method in JavaScript?